src/GovUk.Education.ExploreEducationStatistics.Content.Model/Versioned.cs

```diff
@@ -1,14 +1,19 @@
-using System.Collections.Generic;
+using System;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.Model
 {
-    public interface IVersioned<T> where T : IVersion
+    public abstract class Versioned<TSelf>
     {
-        List<T> Versions { get; set; }
-
-        T Latest { get; }
-
-        T Current { get; }
+        public DateTime Created { get; set; }
+        
+        public User CreatedBy { get; set; } 
+        
+        public Guid CreatedById { get; set; }
+        
+        public TSelf Original { get; set; }
 
+        public Guid OriginalId { get; set; }
```

Comment:
Should this be optional?
I can see the seeded Releases will be getting an Id of new Guid("00000000-0000-0000-0000-000000000000").

Reply:
Good spot, but I will set the correct value for the Releases in ContentDbContext and make sure that that makes it through into the Snapshot file too.

Reply:
Added new migration to updated these